sklearn/base.py

```diff
@@ -322,7 +320,10 @@ def __setstate__(self, state):
                 warnings.warn(
                     "Trying to unpickle estimator {0} from version {1} when "
                     "using version {2}. This might lead to breaking code or "
-                    "invalid results. Use at your own risk.".format(
+                    "invalid results. Use at your own risk. "
+                    "For more info please refer to:\n\n"
+                    "https://scikit-learn.org/stable/modules/model_persistence"
+                    ".html#security-maintainability-limitations \n".format(
```

Comment:
Nit: I do not think we need the newline at the end

```suggestion
                    ".html#security-maintainability-limitations".format(
```

Reply:
I have made the changes you suggested!

Yes there are unrelated changes and I have linting problems with the PR, I don't know why, I'm using black as scikit-ĺearn suggest in the contribution guide.